README.md

```diff
@@ -17,108 +17,16 @@ There is a tutorial [here](examples/tutorial/TUTORIAL.md)
 Building and testing
 --------------------
 
-If you want to use `stack` on MacOS, you will need to have GRPC installed
-already -- see the [Installing GRPC for `stack`](#stackgrpc) section below. Any
-example build and test recipes below which use `stack` assume you have already
-performed the steps described in that section.
-
-Without `stack`, `nix-build release.nix -A grpc-haskell` will build and test the
-whole thing and put the completed package into the nix store. `nix-shell` can be
-used to give you a development environment where you can use the `cabal` and
-`stack` toolchains for development and testing:
+`nix-build release.nix -A grpc-haskell` will build and test the whole thing and
+put the completed package into the nix store. `nix-shell` can be used to give
+you a development environment where you can use `cabal` for development and
+testing:
 
 ```bash
-$ nix-shell release.nix -A grpc-haskell.env
+$ nix-shell
 [nix-shell]$ cabal configure --enable-tests && cabal build && cabal test
 ```
 
-```bash
-$ nix-shell release.nix -A grpc-haskell.env
-[nix-shell]$ stack build --fast && stack test --fast
-```
-
-Note that, for `stack`, the `nix-shell` environment is only needed to run the
-tests, because it uses some custom python tooling (for grpc interop
-testing). You should still be able to `stack build` without using the
-`nix-shell` environment at all.
-
-NB: You can also instruct `stack` to run the tests inside the `nix-shell`
-environment directly, via `stack --nix test --fast`. However, this will
-frequently rebuild the custom ghc that is used in `release.nix`, so is not
-recommended during develop-debug cycles (use the `cabal` path for that, or
-iterate within a `nix-shell`).
-
-Finally, since `stack` does not use `nix` for any Haskell package dependencies,
-be sure to update repository references for dependent packages such as
-`protobuf-wire` in both `nix/<pkg>.nix` AND in `stack.yaml`.
-
-<a name="stackgrpc"></a>Installing GRPC for `stack` (MacOS)
------------------------------------------------------------
-
-If you want to use `stack` in a relatively natural and painless manner, you will
-need a working installation of the GRPC C core libraries.
-
-On MacOS, because of
-issues [related](https://github.com/commercialhaskell/stack/issues/1161) to
-System Integrity Protection, dependencies on the nix-built `grpc` don't seem to
-work properly in the stack toolflow when `DYLD_LIBRARY_PATH` refers to the
-`nix`-built `grpc` library in the nix store, so the library needs to be
-installed somewhere that the loader can pick it up without `DYLD_LIBRARY_PATH`
-set (e.g., in `/usr/local/lib`).
```

Comment:
I think this note about `grpc` from Nixpkgs on macOS is still relevant, no?

Reply:
@evanrelf: Not at as far as I can tell.  I've been doing `gRPC-haskell` development on OS X without issue on `master`

Reply:
Okay cool, that's good to hear